components/list/list-item-drag-drop-mixin.js

```diff
@@ -263,6 +263,11 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 			 * @type {string}
 			 */
 			dragHandleText: { type: String, attribute: 'drag-handle-text' },
+			/**
+			 * **Drag & drop:** Whether the items can be dropped as nested children
+			 * @type {boolean}
+			 */
+			dropNested: { type: Boolean, attribute: 'drop-nested' },
```

Comment:
This new property enables the consumer to indicate which items can become parents by dropping items on them.  This is necessary because not all consumers will want to enable dropping on items to create nested lists, and we also need to ensure that existing lists do not suddenly allow this.